pkg/argo/fetch.go

```
@@ -111,15 +118,54 @@ func checkNodeType(expected, actual wfv1.NodeType) error {
 	return nil
 }
 
-func getPodLogs(podName string) (string, error) {
+func parseOffset(content string) (string, string) {
```

To make debugging easier later on, please return an error on parsing failure.